images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1

```diff
@@ -48,7 +48,7 @@ $markdown += New-MDList -Style Unordered -Lines (@(
     (Get-ChocoVersion),
     (Get-ComposerVersion),
     (Get-HelmVersion),
-    (Get-CondaVersion),
+    "$(Get-CondaVersion) (pre-installed on the image but not added to PATH)",
```

Comment:
maybe add it to the function output instead of the generator?

Reply:
Done. Thank you.